TAO/tests/Compression/RLECompressorTest.cpp

```diff
@@ -9,10 +9,9 @@
 
 #include <memory>
 
-// Older versions of GCC do not support std::unique_ptr!
-struct ACE_Byte_Array_ptr : std::auto_ptr<ACE_Byte> {
+struct ACE_Byte_Array_ptr : std::unique_ptr<ACE_Byte> {
```

Comment:
This struct looks to be something we should completely remove and use a different construct in the code that uses this struct

Reply:
I do not know this code, it was not my intention to refactor ACE_TAO tests at all.

Reply:
At least this should be a new issue, this is a workaround for an older compiler which should be removed

Reply:
Can you please open a Ticket, I am on Holiday next 2 weeks.